lib/tapioca/cli.rb

```diff
@@ -3,6 +3,8 @@
 
 require 'thor'
 
+require_relative "generator"
```

Comment:
With the above change, this shouldn't be needed.

Reply:
This made me think, shouldn't the tests for `cli` and `compilers` only import their respective module instead of `tapioca` all in one?

Reply:
Yeah, not sure about that either. But from what I see almost all gems end up doing all their `require`s in their entrypoint and don't have any other `require` calls anywhere else. Thus, the best practice ends up being to `require` the entrypoint file as it would be in real usage.

Reply:
Fixed too